runtime/src/plugin.rs

```diff
@@ -378,22 +401,22 @@ impl Plugin {
     }
 
     pub(crate) fn initialize_runtime(&mut self) -> Result<(), Error> {
```

Comment:
does this need to be done in kernel-based SDKs?

Reply:
For now yeah, that just makes it possible to side-step `_start` in modules that require initialization.

Reply:
Ok, good to know -- I think we need to include it in TODOs in the `go-sdk` cc/ @mhmd-azeez, unless its already being handled!

Reply:
I think we can use `ModuleConfig.WithStartFunctions` to do the same thing in wazero

Reply:
I added a TODO in the PR readme for it so that we don't forget it